package.json

```diff
@@ -72,6 +72,7 @@
     "@babel/preset-typescript": "^7.22.5",
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-node-resolve": "^13.3.0",
+    "@standard-schema/spec": "^1.0.0-beta.3",
```

Comment:
This is a dev dependency that is only used in tests to make sure we are implementing the schema correctly.